muse/utils.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+
+from glob import glob
+import os
+from collections import OrderedDict
+
+from mne import create_info, concatenate_raws
+from mne.io import RawArray
+from mne.channels import read_montage
+import pandas as pd
+import numpy as np
+import seaborn as sns
+from matplotlib import pyplot as plt
+
+
+sns.set_context('talk')
+sns.set_style('white')
+
+
+def load_muse_csv_as_raw(data_dir, subject_nb=1, session_nb=1, sfreq=256.,
```

Comment:
Can you make an extra method that takes an absolute path to a file and return the raw.